addons/payment_stripe_sca/models/payment.py

```diff
@@ -67,7 +67,8 @@ def _create_stripe_session(self, kwargs):
                 .sudo()
                 .search([("reference", "=", kwargs["client_reference_id"])])
             )
-            tx.stripe_payment_intent = resp["payment_intent"]
+            if tx:  # @dbo we need to setup a transation at test_stripe.test_20_stripe_form_render
```

Comment:
Hello @dbo-odoo , `tx` is an empty record set. There is no transaction created by `test_20_stripe_form_render` but I don't know how to create one so it doesn't fail here.